Documentation/CrossPlatform/LeapMotionMRTK.md

```diff
@@ -4,19 +4,26 @@ A [Leap Motion Controller](https://www.ultraleap.com/product/leap-motion-control
 
 The Leap Motion Data Provider enables articulated hand tracking for VR and could be useful for rapid prototyping in the editor.  The data provider can be configured to use the leap motion controller mounted on a headset or placed on a desk face up.
 
+![LeapMotionIntroGif](../Images/CrossPlatform/LeapMotion/LeapHandsGif3.gif)
+
 ## Using Leap Motion tracking in MRTK
 1. Prepare MRTK project for Leap Motion
 
-    - **This step only applies if the source of MRTK is cloned from the git repo and NOT the from the Unity packages. If the MRTK source is from the Unity packages, go to step 2.**
+    - **This step only applies for the Leap Motion Core Assets version 4.4.0 and if the source of MRTK is cloned from the git repo, NOT from the Unity packages. This step is not required if the Leap Motion Core Assets version 4.5.0 are used. If the MRTK source is going to be from the Unity packages, start at step 2.**
```

Comment:
I guess the neat thing is that even if they do this, it's a no-op from their perspective right?

Reply:
i.e. it adds something to CSC but it doesn't really end up affecting them.

Reply:
Yes, nothing will happen to the project if they do update the csc.  But if someone is using the 4.4.0 core assets then the step is required.